### ISSUE TITLE
dokku renew and enable doest work

### ISSUE THREAD

**Issue by AlexParamonov** [dokku/dokku-letsencrypt#218]
### How reproducible
```
# dokku --version
dokku version 0.23.1
# dokku letsencrypt:enable myapp
=====> Let's Encrypt letsencrypt:enable
 !     App name must begin with lowercase alphanumeric character, and cannot include uppercase characters, colons, or underscores
# dokku letsencrypt:list
-----> App name           Certificate Expiry        Time before expiry        Time before renewal      
myapp                     2021-03-30 00:01:26       6d, 21h, 13m, 32s         23d, 2h, 46m, 28s ago    
# dokku letsencrypt:disable myapp
/var/lib/dokku/plugins/enabled/letsencrypt/subcommands/disable: line 40: letsencrypt_enable_cmd: command not found
# dokku letsencrypt:auto-renew
=====> Auto-renewing all apps...
-----> myapp needs renewal
=====> Let's Encrypt letsencrypt:enable
 !     App name must begin with lowercase alphanumeric character, and cannot include uppercase characters, colons, or underscores
```

#### Actual Results
disable and enable fails with an error

#### Expected Results
It is possible to disable or enable without an error

### How (deb/make/rpm) and where (AWS, VirtualBox, physical, etc.) was Dokku installed?:
apt

I ran cleanup, removed and reinstalled the plugin, but something seems to be broken.

**Comment by MikeAliG** [comment]
Same here. I'm getting:

 !     App name must begin with lowercase alphanumeric character, and cannot include uppercase characters, colons, or underscores

Whenever I want to enable this plugin for an app.

**Comment by josegonzalez** [comment]
Taking a look right now.

**Comment by josegonzalez** [comment]
Should be fixed in 0.10.1. Can someone update their  plugin install and verify that is the case?

**Comment by AlexParamonov** [comment]
it works now, thanks!
